packages/client/ui/react-ui/src/providers/CrossmintAuthProvider.tsx

```diff
@@ -0,0 +1,56 @@
+import { createContext, useEffect, useMemo, useState } from "react";
+
+import { AuthProvider, useAuth as useAuthCore } from "@crossmint/client-sdk-auth-core";
+import type { AuthProviderParams } from "@crossmint/client-sdk-auth-core/src/contexts/authlogin";
+import { EVMSmartWallet, SmartWalletSDK } from "@crossmint/client-sdk-smart-wallet";
+
+export const WalletContext = createContext<{ wallet: EVMSmartWallet | null; isLoadingWallet: boolean }>({
+    wallet: null,
+    isLoadingWallet: false,
```

Comment:
To conform to the PRD now, I think we should drop this. Within the demo we can check `jwt != null && wallet == null` as a replacement